internal/commands/sbomtest/presenter.go

```diff
@@ -127,11 +127,19 @@ func RenderPrettyResult(w io.Writer, orgID, filepath string, res *snykclient.SBO
 	untested := make([]view.Component, 0, len(res.Summary.Untested))
 
 	for i := range res.Vulnerabilities {
+		introducedBy := make([]*view.IntroducedBy, 0, len(res.Vulnerabilities[i].Packages))
```

Comment:
I would rather remove the pointer semantics, and make it a `[]view.IntroducedBy`